client/app/components/Select.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useRef, useState, useEffect } from "react";
+import { sortBy, get } from "lodash";
+import AntdSelect, { SelectProps } from "antd/lib/select";
+
+function getItemOfPercentile<T>(list: Array<T>, percentile: number, sortIteratee = "length") {
+  if (get(list[0], sortIteratee, null) === null) {
+    return;
+  }
+  const sortedList = sortBy(list, sortIteratee);
+  const percentileIndex = Math.ceil((list.length - 1) * (percentile / 100));
+  return sortedList[percentileIndex];
+}
+
+const FONT_RATIO = 0.7;
+
+function Select({ style, options, ...props }: SelectProps<any>): JSX.Element {
```

Comment:
Since this component is not as generic as Ant's Select - it should have more specific name